### ISSUE TITLE
I use destiny item manager, I want to share items from 2 accounts

### ISSUE THREAD

**Issue by shadowwdragonn** [DestinyItemManager/DIM#10803]
### Proposed change

I use destiny item manager, I want to share items from 2 separate accounts, is this possible from steam to epic?

### How does this fit into your workflow?

destiny item manager

**Comment by bhollis** [comment]
Yep you can use two different profiles in your browser and open a different account in each.

**Comment by shadowwdragonn** [comment]
> Yep you can use two different profiles in your browser and open a different account in each.



> Yep you can use two different profiles in your browser and open a different account in each.

I want to transfer items between them

**Comment by bhollis** [comment]
Ah, that's not a thing you can do.
